### PR TITLE
Fix: PrimitiveMesh

### PR DESCRIPTION
**Pull request by singlecoder**
### Please check if the PR fulfills these requirements

- [ ] The commit message follows our [guidelines](https://github.com/oasis-engine/engine/blob/main/.github/COMMIT_MESSAGE_CONVENTION.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

### What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
Fix vertex count over the limit of uint16

### CODE REVIEW

**Review by zhuxudong — APPROVED** [review]
+1

File: packages/core/src/mesh/PrimitiveMesh.ts [hunk 1]
```diff
@@ -6,11 +6,15 @@ import { IndexFormat } from "../graphic/enums/IndexFormat";
...
 import { Mesh } from "../graphic/Mesh";
 import { VertexElement } from "../graphic/VertexElement";
+import { GLCapabilityType } from "../base/Constant";
 
 /**
  * Used to generate common primitve meshes.
  */
 export class PrimitiveMesh {
+  /** The max number of indices that Uint16Array can support. */
+  private static _indiceLimit: number = 65535;
```
Comment (GuoLei1990) [inline packages/core/src/mesh/PrimitiveMesh.ts:16]: this limit count is not index count, is vertex limit Count. Use _uint16VertexLimit or _uint16MaxValue

Comment (singlecoder) [reply packages/core/src/mesh/PrimitiveMesh.ts]: > this limit count is not index count, is vertex limit Count. Use _uint16VertexLimit or _uint16MaxValue

done

**Review by GuoLei1990 — APPROVED** [review]
+1

**Pull request merged**